frontend-web/webclient/app/Dashboard/Dashboard.tsx

```diff
@@ -364,7 +361,6 @@ function DashboardResources({products}: {
         if (compare !== 0) return compare;
 
         compare = a.productType.localeCompare(b.productType);
-        if (compare !== 0) return compare;
```

Comment:
I might be because of the diff-viewer, but if this line is removed, the one above should be as well, I think.